.cirrus.yml

```diff
@@ -61,7 +61,9 @@ validate_task:
         image: "${SKOPEO_CI_CONTAINER_FQIN}"
         cpu: 4
         memory: 8
-    script: make validate-local
+    script: |
+        make validate-local
+        make vendor && hack/tree_status.sh
```

Comment:
Does this need `set -e`, or splitting into two `script` instructions? It’s not obvious to me from the CI log.

Reply:
Nope, not needed.  They do a `set -eo pipefail` by default for bash.

Reply:
Perfect, LGTM. Thanks again!